portfolio/src/main/webapp/style.css

```diff
@@ -1,152 +1,141 @@
+  padding-top: 5%;
+  padding: 8px 0;
+  z-index: 10
 }
- 
 
- 
 .navbar .navbar-header,
 .navbar-collapse {
-    float: none;
-    display: inline-block;
-    vertical-align: top;
-    padding-left: 0;
-    padding-right: 0
+  float: none;
+  display: inline-block;
+  vertical-align: top;
+  padding-left: 0;
+  padding-right: 0
 }
- 
+
 .navbar-nav {
-    float: none;
-    color: #fff
+  float: none;
+  color: #fff
 }
- 
+
 .navbar-default .navbar-nav>li>a {
-    color: #fff
+  color: #fff
 }
- 
- 
-.bitmoji-top {
-    text-align: center;
-    color: #f8f8f8;
-    background: #ec6f6d
 
+.bitmoji-top {
+  text-align: center;
+  color: #f8f8f8;
+  background: #ec6f6d
 }
- 
+
 .bio {
-    text-align: center;
-    padding-top: 5%;
-    padding-bottom: 10;
+  text-align: center;
+  padding-top: 5%;
+  padding-bottom: 10;
 }
-.top-bio {
-    
-    line-height: 32px;
-    text-align: center;
-    padding-top: 0%;
-    padding-bottom: 0;
-
 
+.top-bio {
+  line-height: 32px;
+  text-align: center;
+  padding-top: 0%;
+  padding-bottom: 0;
 }
 
 .about {
-    text-align: center;
-
+  text-align: center;
 }
-.image {
-    padding-top: 5%;
-    padding-bottom: 5%;
-
-    text-align: center;
-    position: relative;
 
+.image {
+  padding-top: 5%;
+  padding-bottom: 5%;
+  text-align: center;
+  position: relative;
 }
- 
+
 .bio>h1 {
-    text-align: center;
-    margin: 0;
-    font-size: 6em;
-    text-shadow: 0 1px 0 rgba(0, 0, 0, .2);
-    font-weight: 700
+  text-align: center;
+  margin: 0;
+  font-size: 6em;
+  text-shadow: 0 1px 0 rgba(0, 0, 0, .2);
+  font-weight: 700
 }
- 
+
 .bio>h3 {
-    text-align: center;
-    text-shadow: 0 1px 0 rgba(0, 0, 0, .2);
-    font-weight: 400;
-    margin-top: 10px
+  text-align: center;
+  text-shadow: 0 1px 0 rgba(0, 0, 0, .2);
+  font-weight: 400;
+  margin-top: 10px
 }
- 
+
 .bio>p {
-    text-align: center;
-    font-size: 20px;
-    line-height: 22px;
-    color: #fff
+  text-align: center;
+  font-size: 20px;
+  line-height: 22px;
+  color: #fff
 }
- 
+
 .info-row {
-    text-align: center;
-    background: #fff;
-    padding: 15px 0
+  text-align: center;
+  background: #fff;
+  padding: 15px 0
 }
- 
+
 .info-row1 {
-    text-align: center;
-    background: #ec6f6d
+  text-align: center;
+  background: #ec6f6d
 }
- 
+
 .info-row h2 {
-    text-align: center;
-    color: #000;
-    margin-bottom: 30px
+  text-align: center;
+  color: #000;
+  margin-bottom: 30px
 }
- 
+
 .info-row1 h2 {
-    text-align: center;
-    color: #fff;
-    margin-bottom: 30px
+  text-align: center;
+  color: #fff;
+  margin-bottom: 30px
 }
- 
+
 .info-row p {
-    text-align: center;
-    color: #000;
-    font-size: 20px;
-    line-height: 32px
+  text-align: center;
+  color: #000;
+  font-size: 20px;
+  line-height: 32px
 }
- 
+
 .info-row1 p {
-    text-align: center;
-    color: #fff;
-    font-size: 20px;
-    line-height: 32px
-}
+  text-align: center;
+  color: #fff;
+  font-size: 20px;
+  line-height: 32px
+  
+}
```

Comment:
nit: please add an empty line at the end (you can see that one is missing when there is the red sign at the end of the file)